public/js/user/home.js

```diff
@@ -0,0 +1,32 @@
+const formContactUs = document.querySelector('.footer--contactUs-form');
+const registerEmail = document.querySelector('.footer--form');
+
+formContactUs.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const formData = new FormData(formContactUs);
+  const dataContactUs = {};
+  formData.forEach((value, key) => {
+    dataContactUs[key] = value;
+  });
+  fetch('/contact-us', {
+    method: 'POST',
+    body: JSON.stringify(dataContactUs),
+    headers: { 'Content-Type': 'application/json' },
+  })
+    .then(res => console.log(res.json()));
+});
+
+registerEmail.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const formData = new FormData(registerEmail);
+  const dataregisterEmail = {};
+  formData.forEach((value, key) => {
+    dataregisterEmail[key] = value;
+  });
+  fetch('/register', {
+    method: 'POST',
+    body: JSON.stringify(dataregisterEmail),
+    headers: { 'Content-Type': 'application/json' },
+  })
+    .then(res => console.log(res.json()));
```

Comment:
Remove console logs.

Reply:
i'll do that after database done cuz no error show while i'm do the dom and css files